src/static/js/common_tableform.js

```diff
@@ -2002,8 +2002,9 @@ const tableform = {
                 // Unescaped tags in textareas behave unpredictably
                 var s = row[v.json_field];
                 if (!s) { s = ""; }
-                s = s.replace(/</g, "&lt;").replace(/>/g, "&gt;");
-                n.val(html.decode(s));
+                n.textarea("value", s);
+                n.textarea('process_tokens');
+                n.change();
```

Comment:
can't process_tokens call change if necessary? Also, for consistency I always use double quotes when quoting strings, the only time I use single quotes is if the string contains HTML, so I can use double quotes for HTML attributes